gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -107,6 +112,26 @@ export default function NxFormLayoutExample() {
           </select>
         </label>
       </div>
+      <fieldset className="nx-fieldset">
+        <legend className="nx-legend">
+          <span className="nx-legend__text">
+            Here's what you can do: {isWarpOn && 'Warp drive started'}
+            {isKrakenOut && ' The Kraken is out!'} {isShapes && ' Shapes are allowed'}
+          </span>
+        </legend>
+        <span className="nx-sub-label">
```

Comment:
Based on the other examples this should be inside of the `legend`.

Reply:
> Based on the other examples this should be inside of the `legend`.

Nice that it still works outside though: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/ab78e6c29549ead547fdc4d8df713039c841aad8